src/main/java/oracle/kubernetes/operator/rest/backend/RestBackend.java

```diff
@@ -63,4 +65,16 @@
    * @param managedServerCount - the desired number of WebLogic managed servers.
    */
   public void scaleCluster(String domainUID, String cluster, int managedServerCount);
+
+
+  /**
+   * Upgrade applications
+   * @param domainUID - the unique identifier assigned to a WebLogic domain
+   * when it was registered with the WebLogic operator.  The caller is responsible
+   * for calling isDomainUID first and not calling this method if the domain has not
+   * been registered.
+   * @param appsToUpgrade - Containing info of the application to be upgraded.
+   *                        It's provided as REST API input data.
```

Comment:
What does this mean?

Reply:
Change it to:
   * @param appsToUpgrade - Containing application info that is provided by the structure data
   *                      when posting an REST request
Hope this is clearer.